netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +159,31 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages has to throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test(expected = HttpException.class)
+    public void exceptionIfBoutNotFound() throws IOException {
+        Messages messages = null;
+        try {
+            final String name = "testunread";
+            final Aliases aliases =
+                new DyBase().user(new URN("urn:test:75066")).aliases();
+            final Alias alias = aliases.add(name);
+            final Inbox inbox = alias.inbox();
+            final Bout bout = inbox.bout(inbox.start());
+            bout.messages().post("    4 leading spaces retained  ");
+            final Friends friends = bout.friends();
+            friends.kick(name);
+            messages = bout.messages();
```

Comment:
@2686747 This is because `DyBase` is a closable. Please initialize it in a try-with-resources and run the actual test code inside that try-with-resources. Otherwise you test case might also be leaking resources (In fact I think it is) because `DyBase` is a `Closable`.
Just handle everything inside one try-with-resources block and we're good here imo.